src/components/organelle-wrapper.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+declare var Organelle: any;
+
+interface OrganelleWrapper {
```

Comment:
I think this is the TypeScript way to check prop types

Reply:
If I go in and change all the types of the props in this interface to be other things, I don't see any errors either in the code editor or when building the project. I don't even see console errors. So I'm not sure it's reading this, or I'm missing something.